tests/neg/i15784.check

```diff
@@ -10,17 +10,17 @@
   |                      Not found: A
   |
   | longer explanation available when compiling with `-explain`
--- Warning: tests/neg/i15784.scala:7:8 ---------------------------------------------------------------------------------
+-- Warning: tests/neg/i15784.scala:7:7 ---------------------------------------------------------------------------------
 7 |  case X: Int => X  // warn
-  |        ^
-  |        Type ascriptions after patterns other than:
-  |          * variable pattern, e.g. `case x: String =>`
-  |          * number literal pattern, e.g. `case 10.5: Double =>`
-  |        are no longer supported. Remove the type ascription or move it to a separate variable pattern.
--- Warning: tests/neg/i15784.scala:10:12 -------------------------------------------------------------------------------
+  |       ^
+  |       Type ascriptions after patterns other than:
+  |         * variable pattern, e.g. `case x: String =>`
+  |         * number literal pattern, e.g. `case 10.5: Double =>`
+  |       are no longer supported. Remove the type ascription or move it to a separate variable pattern.
+-- Warning: tests/neg/i15784.scala:10:7 --------------------------------------------------------------------------------
 10 |  case `Int`: Int => `Int` // warn
-   |            ^
-   |            Type ascriptions after patterns other than:
-   |              * variable pattern, e.g. `case x: String =>`
-   |              * number literal pattern, e.g. `case 10.5: Double =>`
-   |            are no longer supported. Remove the type ascription or move it to a separate variable pattern.
+   |       ^^^^^
+   |       Type ascriptions after patterns other than:
+   |         * variable pattern, e.g. `case x: String =>`
+   |         * number literal pattern, e.g. `case 10.5: Double =>`
+   |       are no longer supported. Remove the type ascription or move it to a separate variable pattern.
```

Comment:
Sorry I didn't notice it before; I was distracted by the long message and the fact that Scala 2 doesn't support it.

One could misquote the saying about seeing the caret in one's brother's eye.